web3/__init__.py

```diff
@@ -45,7 +45,6 @@
     "HTTPProvider",
     "IPCProvider",
     "WebsocketProvider",
-    "TestRPCProvider",
```

Comment:
This doesn't actually point to anything or work for me locally, does it for you? It doesn't point to anything that still exists and `from web3 import *` doesn't actually pull in `TestRPCProvider` in a fresh virtual env.

I'm pretty sure this could go on both `v5` and `v6`. But if we only merge this to `v6` then this could be updated separately in `v5`.